src/rednoise.c

```diff
@@ -5,10 +5,17 @@
 #include "dmalloc.h"
```

Comment:
So did you not need that #include "misc_utils.h" after all?  Probably not since it is already included in "presto.h".  Yeah, I think that's right.